apollo-api/src/main/java/com/apollographql/android/api/graphql/Response.java

```diff
@@ -0,0 +1,28 @@
+package com.apollographql.android.api.graphql;
+
+import java.util.List;
+
+import javax.annotation.Nullable;
+
+/** Represents either a successful or failed response received from the GraphQL server. */
+public class Response<T extends com.apollographql.android.api.graphql.Operation.Data> {
+  @Nullable private final T data;
+  @Nullable private final List<com.apollographql.android.api.graphql.Error> errors;
+
+  public Response(@Nullable T data, @Nullable List<com.apollographql.android.api.graphql.Error> errors) {
```

Comment:
whats up with this class being recreated both prs?  Also is the fully qualified name in `List<>` what you expected?